examples/webvr_stereo_pano.html

```diff
@@ -59,7 +59,7 @@
 			effect = new THREE.VREffect( renderer );
 
 
-			if ( navigator.getVRDevices ) {
+			if ( navigator.getVRDisplays || navigator.getVRDevices ) {
```

Comment:
it looks like there's [a bit more](https://github.com/toji/three.js/blob/webvr_v1/examples/webvr_stereo_pano.html#L214) that has to be done with this example to make it work with the new API.